Mathlib/MeasureTheory/Measure/AddContent.lean

```diff
@@ -0,0 +1,185 @@
+/-
+Copyright (c) 2024 Rémy Degenne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Rémy Degenne, Peter Pfaffelhuber
+-/
+import Mathlib.MeasureTheory.SetSemiring
+
+/-!
+# Additive Contents
+
+An additive content `m` on a set of sets `C` is a set function with value 0 at the empty set which
+is finitely additive on `C`. That means that for any finset `I` of pairwise disjoint sets in `C`
+such that `⋃₀ I ∈ C`, `m (⋃₀ I) = ∑ s ∈ I, m s`.
+
+Mathlib also has a definition of contents over compact sets: see `MeasureTheory.Content`.
+A `Content` is in particular an `AddContent` on the set of compact sets.
```

Comment:
What's the long-term goal here? Are you planning to redefine `Content` using `AddContent`? Can you record that as a TODO?